DataQualityTester/templates/partials/_test_another_file.html

```diff
@@ -2,5 +2,8 @@
   <div class="col-lg-2">
     <a href="{{ url_for('home') }}" class="btn btn-default">Test another file</a>
   </div>
+  <div class="col-lg-2">
+    <a href="{{ url_for('download_results', uuid=uuid) }}" class="btn btn-default">Download results</a>
+  </div>
   <div class="col-lg-offset-5 col-lg-5"></div>
 </div>
```

Comment:
A nice to have would be for this column layout to be fixed. The offset-5  div is a bit weird, you'd also think that would need decrementing, but also there are other strange things going on with padding/margins to stop the buttons from overlapping and that is why they sit so far apart. Kind of odd, unfortunately forgotten most of my bootstrap 3 knowledge.